src/DblParser/Main.ml

```diff
@@ -35,7 +35,8 @@ let rec repl_seq imported () =
     Seq.Cons(def, repl_seq imported)
 
   | Raw.REPL_Def def ->
-    Seq.Cons(Desugar.tr_def def, repl_seq imported)
+    let defs = Desugar.tr_def def |> List.to_seq in
+    (Seq.append defs (repl_seq imported)) ()
```

Comment:
You should be careful with this line, when fixing merge conflicts. It should not provide a bug, but may brake the invariant mentioned in https://github.com/fram-lang/dbl/pull/90#pullrequestreview-2041218888, which protects us from bugs similar to #83.